### ISSUE TITLE
NBDime isn't compatible with JupyterLab 1.0.0

### ISSUE THREAD

**Issue by ZainRizvi** [jupyter/nbdime#485]
When trying to install NBDime on JupyterLab >= 1.0.0 I get the error:
```

The extension "nbdime-jupyterlab" is outdated.

error occured.
ValueError: This extension does not yet support the current version of JupyterLab.

Conflicting Dependencies:
JupyterLab              Extension      Package
>=3.0.0 <4.0.0          >=2.0.2 <3.0.0 @jupyterlab/coreutils
>=1.0.0 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/application
>=1.0.0 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/console
>=1.0.0 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/filebrowser
>=1.0.0 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/terminal
>=1.0.0 <2.0.0          >=0.19.1 <0.20.0@jupyterlab/apputils
>=4.0.0 <5.0.0          >=3.1.0 <4.0.0 @jupyterlab/services
```

**Comment by bmwilly** [comment]
You don't need to install it with `jupyter labextension`. 

Just do 
```
jupyter labextension uninstall nbdime-jupyterlab
pip install -U nbdime  # or conda
```

**Comment by vidartf** [comment]
Nbdime has not yet been updated to support lab 1.0. Follow #483 to track the progress.

**Comment by vidartf** [comment]
Fixed in #483
